doc/releases/release-notes-4.0.rst

```diff
@@ -316,6 +316,21 @@ Drivers and Sensors
   * Fixed SPI NOR driver issue where wp, hold and reset pins were incorrectly initialized from
     device tee when SFDP at run-time has been enabled (:github:`80383`)
 
+  * Added :kconfig:option:`CONFIG_SPI_NOR_ACTIVE_DWELL_MS`, to SPI NOR driver configuration,
+    that allows to set time for which driver will wait before triggering Deep Power Down (DPD).
+    The option replaces ``CONFIG_SPI_NOR_IDLE_IN_DPD`` aiming at reduction of unneeded power
+    state changes and SPI transfers between other operations, specifically when burst type
+    access to SPI NOR device occurs.
+
+  * Added :kconfig:option:`CONFIG_SPI_NOR_INIT_PRIORITY` that allows to select initialization
+    priority for SPI NOR driver.
+
+  * Flash API has been extended with :c:func:`flash_copy` utility function allowing to perform
+    direct copy of data between two Flash API devices.
+
+  * Fixed Flash Simulator issue where offsets have been assumed as absolute instead of relative
+    to device base (:github:`79082`).
```

Comment:
```suggestion
  * Fixed a Flash Simulator issue where offsets were assumed to be absolute instead of relative
    to the device base address (:github:`79082`).
```